js/hoot/view/utilities/Dataset.js

```diff
@@ -73,10 +73,19 @@ Hoot.view.utilities.dataset = function(context)
             .attr('class', 'label')
             .text(function(d) { return d.title; });
 
-        form.append('div')
+        var table = form.append('div')
         .attr('id','datasettable')
-            .classed('col12 fill-white small strong row10 overflow keyline-all', true)
-            .call(hoot_view_utilities_dataset.populateDatasetsSVG);
+            .classed('col12 fill-white small strong row10 overflow keyline-all', true);
+
+        var datasetColHeaders = [{title: 'Dataset', class: '14%'}, {title: 'Last Accessed', class: '7%'}, {title: 'Import Date', class: '6%'}, {title: 'Size', class: '1%'}];
+        table.insert('div').attr('id', 'datasettableHeader')
+            .append('span').selectAll('th')
```

Comment:
Do we need the span when we already have a div container?

Reply:
I guess not.